nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/AbstractMongoProcessor.java

```diff
@@ -83,15 +78,6 @@ public abstract class AbstractMongoProcessor extends AbstractProcessor {
         .identifiesControllerService(MongoDBClientService.class)
         .build();
 
-    static final PropertyDescriptor URI = new PropertyDescriptor.Builder()
```

Comment:
This removal requires migration, can you add it to the [Migration Guide](https://cwiki.apache.org/confluence/display/NIFI/Migrating+Deprecated+Components+and+Features+for+2.0.0) and/or the appropriate 2.0 docs, since it renders existing flows using these processors invalid? Please and thanks!